src/uct/api/v2/uct_v2.h

```diff
@@ -341,6 +341,22 @@ typedef enum {
 } uct_ep_connect_to_ep_param_field_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief field mask of @ref uct_ep_is_connected_params_t
+ */
+typedef enum {
+    /** Device address */
```

Comment:
minor: these comments are not very explanatory and look like some antipattern. Maybe do not add comments here at all?

Reply:
IMO we do need to keep it, at least for the completeness & consistency of Doxygen output.